pkg/providers/amifamily/resolver.go

```diff
@@ -29,11 +30,16 @@ import (
 
 	"github.com/aws/karpenter-provider-aws/pkg/apis/v1beta1"
 	"github.com/aws/karpenter-provider-aws/pkg/providers/amifamily/bootstrap"
+	"github.com/aws/karpenter-provider-aws/pkg/providers/ssm"
 
 	"sigs.k8s.io/karpenter/pkg/cloudprovider"
 	"sigs.k8s.io/karpenter/pkg/scheduling"
 )
 
+const (
+	AMIVersionLatest = "latest"
```

Comment:
Worth calling out in a function what this refers to. Some users reading this might confuse it as the key used for SSM path discovery, which would be wrong when `latest` == `/recommended`.